scripts/mkimage

```diff
@@ -214,17 +214,27 @@ elif [ "${BOOTLOADER}" = "u-boot" -a -n "${UBOOT_SYSTEM}" ]; then
 
   DTB="$(${SCRIPTS}/uboot_helper ${PROJECT} ${DEVICE} ${UBOOT_SYSTEM} dtb)"
   if [ -n "${DTB}" ]; then
-
-    if [ -f "${RELEASE_DIR}/3rdparty/bootloader/${DTB}" ]; then
-      mcopy "${RELEASE_DIR}/3rdparty/bootloader/${DTB}" ::
+    mkdir -p "${LE_TMP}/dtbs"
+    while IFS= read -r -d '' file; do
+      echo "image: adding device tree file $(basename "$file")"
+      mcopy -n "$file" "${LE_TMP}/dtbs/$(basename "$file")"
+    done < <(find "${RELEASE_DIR}/3rdparty/bootloader/" -maxdepth 1 -name "${DTB}" -print0)
```

Comment:
if you are explicit about the dtb's you don't need to do this.

Reply:
Getting only configured files from array.